docker/Dockerfile

```diff
@@ -57,6 +57,7 @@ RUN if [ "$(uname -m)" != "aarch64" ]; then \
     && \
     rm -rf /var/lib/apt/lists/* /tmp/*
 
+# TODO: this ENV variables are not required at build time, so they're not needed as ARGs either
```

Comment:
The backtrace, colorbt, rust_log, and cargo_home variables are required at build time.